internal/plugin_config_test.go

```
@@ -788,13 +788,9 @@ policies:
 	p := Plugin{}
 
 	err = p.Config([]byte(config), tmpDir)
-	if err == nil {
-		t.Fatal("Expected an error but did not get one")
+	if err != nil {
+		t.Fatal("Did not expect errors but got one")
```

This should also return the error message now that it's no longer `nil`.